Common/XGBoost/src/main/java/org/tribuo/common/xgboost/XGBoostTrainer.java

```diff
@@ -63,7 +63,8 @@
  * </pre>
  * N.B.: This uses a native C implementation of xgboost that links to various C libraries, including libgomp
  * and glibc. If you're running on Alpine, which does not natively use glibc, you'll need to install glibc
- * into the container.
+ * into the container. On Windows this binary is not available in the Maven Central release, you'll need
+ * to compile it from source.
```

Comment:
Do we have any extant advice on compiling XGBoost? Could we link to some tribuo.org page on how to do it? I guess we don't want to support the myriad Windows installs out there ("I couldn't get it to work on Windows ME!")

Reply:
Basically you need to install the Windows SDK to get the msbuild system and then run mvn package in the jvm-packages directory. I can talk to the XGBoost people about it and maybe firm up some instructions.